cookbooks/logstash/recipes/default.rb

```diff
@@ -1,16 +1,37 @@
 # Logstash recipe
 
 # Create directories
-directory "#{node["logstash"]["dir"]}"
-directory "#{node["logstash"]["dir"]}/conf"
-directory "#{node["logstash"]["dir"]}/data"
+directory "#{node["logstash"]["dir"]}" do
```

Comment:
@ervinb, should I use symbols instead of strings for parameters? Chef documentation insists on the latter.
I will refactor this part of the code, thank you. :bear: :ok_hand: